bfe_upload_snapshot.py

```diff
@@ -34,14 +34,17 @@ def main(bf: Session, bf_network: str, snapshot_dir: str) -> None:
     settings = dotenv_values(args.settings)
 
     bf_host = settings.get('BF_HOST', None)
-    bf_port = settings.get('BF_PORT', 443)
+    bf_enterprise = settings.get('BF_ENTERPRISE', "false").lower() == "true"
+    bfe_port = settings.get('BFE_PORT', 443)
     bf_network = settings.get('BF_NETWORK', None)
 
     if bf_host is None:
         raise Exception(f"BF_HOST is not set in {args.settings}")
     if bf_network is None:
         raise Exception(f"BF_NETWORK is not set in {args.settings}")
 
-    bf = Session(host=bf_host, port=bf_port, access_token=args.access_token)
+    print(bf_enterprise)
```

Comment:
I am presuming this is a left over print statement that isn't needed?

Reply:
yes, fixed.